src/storage/src/healthcheck.rs

```diff
@@ -41,9 +41,12 @@ pub async fn write_to_persist(
     let mut handle = client
         .open_writer(
             status_shard,
-            &format!("healthcheck::write_to_persist {}", collection_id),
             Arc::new(relation_desc.clone()),
             Arc::new(UnitSchema),
+            Diagnostics::new(
+                &collection_id.to_string(),
```

Comment:
you need to use the `id` of the `status_history` table here right? this is the id of the table we are writing an status update for

Reply:
good spot! fixed